boat-scaffold/src/main/java/com/backbase/oss/codegen/java/BoatSpringCodeGen.java

```diff
@@ -161,10 +164,11 @@ public void processOpts() {
             this.useSetForUniqueItems = false;
         }
 
-        this.supportingFiles.stream()
-            .filter(sf -> "apiUtil.mustache".equals(sf.templateFile))
-            .findAny()
-            .ifPresent(this.supportingFiles::remove);
+        writePropertyBack("useApiUtil",
```

Comment:
wether it's using ApiUtil or not. Developers can enable / disable it by specifying it in the list of supporting files

Reply:
Exactly, no need to introduce a new configuration